release-please-config.json

```diff
@@ -2,6 +2,7 @@
   "plugins": ["node-workspace"],
   "bump-minor-pre-major": true,
   "bump-patch-for-minor-pre-major": true,
+  "release-as": "21.10.0",
```

Comment:
this is the top level key we set only for new Looker binding's releases (and then remove it after the release-pr is merged)

Reply:
Thanks for the reminder